### ISSUE TITLE
[Request] Add new plants (carrots, tomatoes ...)

### ISSUE THREAD

**Issue by lnjX** [Gamerbude/Minetest_TNG#12]

**Comment by fridolin2** [comment]
![default_strawberry_seed](https://cloud.githubusercontent.com/assets/17991066/14021809/640a5b2a-f1dd-11e5-9d1b-9293a6e58b7c.png)
![default_strawberry_plant_7](https://cloud.githubusercontent.com/assets/17991066/14021811/676463e2-f1dd-11e5-977f-1a9fa1d74cbc.png)
![default_strawberry_plant_6](https://cloud.githubusercontent.com/assets/17991066/14021815/68ad7432-f1dd-11e5-9ab0-01aeeb1a72b5.png)
![default_strawberry_plant_5](https://cloud.githubusercontent.com/assets/17991066/14021816/694479f4-f1dd-11e5-9580-6e3dced1902e.png)
![default_strawberry_plant_4](https://cloud.githubusercontent.com/assets/17991066/14021818/69f2b398-f1dd-11e5-8461-486458b0547c.png)
![default_strawberry_plant_3](https://cloud.githubusercontent.com/assets/17991066/14021822/6c153d26-f1dd-11e5-9546-d05f8aa81721.png)
![default_strawberry_plant_2](https://cloud.githubusercontent.com/assets/17991066/14021824/6d597c38-f1dd-11e5-8467-3ad879c7abd8.png)
![default_strawberry_plant_1](https://cloud.githubusercontent.com/assets/17991066/14021825/6e4962de-f1dd-11e5-8d63-c2bef1877f82.png)
![default_strawberry_item](https://cloud.githubusercontent.com/assets/17991066/14021828/6fc288e8-f1dd-11e5-91ee-09f83529fff8.png)

**Comment by fridolin2** [comment]
![default_cabbage_item](https://cloud.githubusercontent.com/assets/17991066/14022785/a75284bc-f1e1-11e5-859d-2a3d3d532c34.png)
![default_cabbage_plant_1](https://cloud.githubusercontent.com/assets/17991066/14022786/aaaebf90-f1e1-11e5-959e-2a84a0355c53.png)
![default_cabbage_plant_2](https://cloud.githubusercontent.com/assets/17991066/14022791/acfd9f82-f1e1-11e5-93f1-c68ea0a3997f.png)
![default_cabbage_plant_3](https://cloud.githubusercontent.com/assets/17991066/14022793/aefae8ee-f1e1-11e5-93cb-b979745e4666.png)
![default_cabbage_plant_4](https://cloud.githubusercontent.com/assets/17991066/14022797/b19333b8-f1e1-11e5-8dc4-bc7321ab6d2a.png)
![default_cabbage_plant_5](https://cloud.githubusercontent.com/assets/17991066/14022798/b3ec0b44-f1e1-11e5-93e6-8ad2a19ca342.png)

plant five is a joke

**Comment by jbruechert** [comment]
I would like to have corn, but this requires an farming-api change to use visual scale without using minetest.override item.

Textures are already done in my agriculture mod:
![agriculture_corn_seed](https://cloud.githubusercontent.com/assets/13609393/14083186/95c96284-f514-11e5-81a9-070839b33395.png)
![agriculture_corn](https://cloud.githubusercontent.com/assets/13609393/14083179/9595c9ba-f514-11e5-97f7-18bfb679cd9e.png)
![agriculture_corn_1](https://cloud.githubusercontent.com/assets/13609393/14083176/957d3b0c-f514-11e5-99dd-cd47ea6bb872.png)
![agriculture_corn_2](https://cloud.githubusercontent.com/assets/13609393/14083180/959a8a86-f514-11e5-9558-7edd33f18526.png)
![agriculture_corn_3](https://cloud.githubusercontent.com/assets/13609393/14083178/958cf1f0-f514-11e5-805a-6312058bd86b.png)
![agriculture_corn_4](https://cloud.githubusercontent.com/assets/13609393/14083177/958606e2-f514-11e5-9c74-24b24c62959b.png)
![agriculture_corn_5](https://cloud.githubusercontent.com/assets/13609393/14083181/95a1a4e2-f514-11e5-80fa-599ad335e2d4.png)
![agriculture_corn_6](https://cloud.githubusercontent.com/assets/13609393/14083182/95a9765e-f514-11e5-92a3-ca35b214b4dc.png)
![agriculture_corn_7](https://cloud.githubusercontent.com/assets/13609393/14083183/95b561bc-f514-11e5-836b-7605db26d666.png)
![agriculture_corn_8](https://cloud.githubusercontent.com/assets/13609393/14083184/95bbe65e-f514-11e5-9de4-0c4b3d2c46db.png)

the use of corn would be:
![agriculture_corn_baken](https://cloud.githubusercontent.com/assets/13609393/14083185/95c5366e-f514-11e5-876e-27eb25c95214.png)
(baken corn)

or corn bread:
![agriculture_corn_bread](https://cloud.githubusercontent.com/assets/13609393/14083249/e37d9df6-f514-11e5-8a53-b79d1c91216c.png)

**Comment by jbruechert** [comment]
For adding cakes sugar beets would be good:
![agriculture_sugar_beet](https://cloud.githubusercontent.com/assets/13609393/14083340/7682c55e-f515-11e5-9f3f-d1fd45a59ce4.png)
![agriculture_sugar_beet_1](https://cloud.githubusercontent.com/assets/13609393/14083344/76a5da8a-f515-11e5-9cfb-abc5c6962083.png)
![agriculture_sugar_beet_2](https://cloud.githubusercontent.com/assets/13609393/14083341/7693a482-f515-11e5-98e8-b2c84c11668e.png)
![agriculture_sugar_beet_3](https://cloud.githubusercontent.com/assets/13609393/14083345/76ac840c-f515-11e5-8bb9-f20ed9839097.png)
![agriculture_sugar_beet_4](https://cloud.githubusercontent.com/assets/13609393/14083342/769bf98e-f515-11e5-96f1-e1751fede286.png)
![agriculture_sugar_beet_5](https://cloud.githubusercontent.com/assets/13609393/14083343/769f7ad2-f515-11e5-984e-4568a4f39c82.png)

**Comment by LNJ2** [comment]
Yeah, maybe. I have already begun to rewrite the API, so this won't be a problem.

**Comment by lnjX** [comment]
I found another farming mod that contains a lot of textures, maybe we can use some of them. 
https://github.com/tenplus1/farming/tree/master/textures

**Comment by jbruechert** [comment]
Because we plan to add seasons (spring, summer ...) I started to work on some other leave textures  for each season:
![defaultleavesautumn](https://cloud.githubusercontent.com/assets/13609393/14497830/7cf29f56-0198-11e6-909c-1eb4a558dd34.png)
![defaultmapleautum](https://cloud.githubusercontent.com/assets/13609393/14497829/7cf02bb8-0198-11e6-9c3a-b698d5973998.png)
![defaultmaplemapletestred](https://cloud.githubusercontent.com/assets/13609393/14497828/7ceecf20-0198-11e6-86eb-938e372f705b.png)
![defaultmaplemapletest](https://cloud.githubusercontent.com/assets/13609393/14497832/7d7336b6-0198-11e6-8b5b-040914983d36.png)

Here is a texture for cherry blossoms wich isn't completely ready for now:
![defaultcherrzleaves](https://cloud.githubusercontent.com/assets/13609393/14497831/7cf38ee8-0198-11e6-8d66-7e85af7472d5.png)

**Comment by LNJ2** [comment]
https://github.com/MatrexsVigil/harvestcraft/issues/111
Maybe we can use textures from harvest craft for e.g. lettuce.
